editor/project_manager.cpp

```diff
@@ -2270,6 +2270,9 @@ ProjectManager::ProjectManager() {
 			} break;
 		}
 
+		// Define a minimum window size to prevent UI elements from overlapping or being cut off
+		OS::get_singleton()->set_min_window_size(Size2(750, 420) * EDSCALE);
```

Comment:
In my testing, I could shrink the project manager down to a little under 700x400 pixels before UI elements are cut off. So, this could be reduced to 700x400 if we wish.

Reply:
I added a bit of safety margin in case the user increases the font size :slightly_smiling_face:

We can easily adapt the minimum window size according to the editor scale, but adapting it for the font size is more difficult as not all elements will grow in the same way.